docs/google/devices.md

```diff
@@ -84,6 +84,44 @@ using a error rate of 0.5% per gate, a circuit of depth 20 and width 20 could be
 at 0.995^(20*20) = 0.135. Using separate error rates per gates (i.e. based on calibration
 metrics) or a more complicated noise model can result in more accurate error estimation.
 
+### Use sweeps when possible
+
+Round trip network time to and from the engine typically adds latency on the order of a second
+to the overall computation time.  Reducing the number of trips and allowing the engine to
+properly batch circuits can improve the throughput of your calculations.  One way to do this
+is to use parameter sweeps to send multiple variations of a circuit at once.
+
+One example is to turn single-qubit gates on or off by using parameter sweeps.  
+For instance, the following code demonstrates combining mesauring the |0> and the |1> state
+into one circuit.  This code uses a simulator rather than the engine to demonstrate the concept.
+
+```python
+import cirq
+import sympy
+
+q = cirq.GridQubit(1,1)
+sim = cirq.Simulator()
+
+# Measure |0>
+m0 = cirq.Circuit(cirq.measure(q))
+# Measure |1>
+m1 = cirq.Circuit(cirq.X(q), cirq.measure(q))
+
+# Executing the circuits requires two calls to the simulator (or engine)
+sim.run(m0)
+sim.run(m1)
+
+# Parameterized circuit
+m_parameterized=cirq.Circuit(cirq.X(q) ** sympy.Symbol('e'), cirq.measure(q))
```

Comment:
Nit: "e" is a loaded term. What do you think of "i" to evoke e^pi*i?

Reply:
Changed to use 't'.  I think that 'i' is also a loaded term in the same way that e is.